modules/blockexplorer/data.go

```diff
@@ -0,0 +1,66 @@
+package blockexplorer
+
+import (
+	"errors"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// Calculates the total number of coins that have ever been created
+// from the bockheight
+func (be *BlockExplorer) totalCurrency() types.Currency {
+	totalCoins := uint64(0)
+	coinbase := types.InitialCoinbase
+	minCoinbase := types.MinimumCoinbase
+	for i := types.BlockHeight(0); i < be.blockchainHeight; i++ {
+		totalCoins += coinbase
+		if coinbase > minCoinbase {
+			coinbase--
+		}
+	}
+	return types.NewCurrency64(totalCoins).Mul(types.SiacoinPrecision)
```

Comment:
I find it interesting that you would multiply this value by SiacoinPrecision when that precision is never shown to the user nor used in the calculation